must-gather/common

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+export KEPLER_NS="openshift-kepler-operator"
+export LOGFILE_NAME="gather-debug.log"
+
+get_timestamp(){
+  date '+%Y-%m-%d %H:%M:%S'
+}
+
+log_stdout() {
+  echo "$(get_timestamp) ${*}"
+}
+
+log_file() {
+  echo "$(get_timestamp) ${*}" >> $LOGFILE_PATH
+}
+
+log_stdout_and_file() {
+  echo "$(get_timestamp) ${*}" | tee -a $LOGFILE_PATH
+}
+
+log(){
+  log_stdout_and_file "$@"
+}
+
+
+run() {
+  length=$(($#-1))
+  cmd=${@:1:$length}
+  gather_file="${@: -1}"
+
+  log "$cmd"
+  $cmd >> $gather_file  2>> $LOGFILE_PATH
```

Comment:
could you please run shellcheck on all scripts? I think it should catch a lot of bash gotchas that we overlook during a review. 

suggestion: May be this is a good PR to add shellcheck to our CI as well. Or have that added before this PR is merged since this script will be used in production.

Reply:
I have added https://github.com/sustainable-computing-io/kepler-operator/pull/262 which adds shellcheck and  fixes shellcheck issues in existing scripts.

Reply:
dome shellcheck, and fixed